examples/CantileverDistributedLoad/cantilever_conservative_distributed_load.py

```diff
@@ -0,0 +1,224 @@
+# from charset_normalizer.legacy import ResultDict
+from matplotlib import pyplot as plt
+import numpy as np
+import elastica as ea
+import json
+from cantilever_distrubuted_load_postprecessing import (
+    plot_video_with_surface,
+    find_tip_position,
+    adjust_square_cross_section,
+)
+
+
+def conservative_force_simulator(load, Animation=False):
+    class StretchingBeamSimulator(
```

Comment:
For all `StretchingBeam` / `stretch_sim` / `stretchable_rod` and any other ones that are appropriate, change to something more coherent to this case, e.g. `square_rod`.

Reply:
Please double check whether this is completed. I can see this class is still called `StretchingBeamSimulator`. Also, I recommend being consistent with your naming. For example, just name all simulator variables `square_rod_sim` and all rods `square_rod` across all three cases (you can do a directory-wide find in IDE).